x-pack/legacy/plugins/siem/cypress/integration/lib/inspect/helpers.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { DEFAULT_TIMEOUT } from '../util/helpers';
+
+import { INSPECT_BUTTON_ICON, InspectButtonMetadata } from './selectors';
+
+export const openHostsAndStatsTables = (table: InspectButtonMetadata) => {
+  if (table.tabId) {
+    cy.get(table.tabId).click({ force: true });
+  }
+  cy.get(table.id, { timeout: DEFAULT_TIMEOUT });
+  if (table.altInspectId) {
+    cy.get(table.altInspectId, { timeout: DEFAULT_TIMEOUT }).trigger('click', {
+      force: true,
+    });
+  } else {
+    cy.get(`${table.id} ${INSPECT_BUTTON_ICON}`, {
+      timeout: DEFAULT_TIMEOUT,
+    }).trigger('click', { force: true });
+  }
+};
+
+export const closesModal = () => {
+  cy.get('[data-test-subj="modal-inspect-close"]').click();
```

Comment:
The `afterEach` that invoked this was flaking out on me, perhaps due to the shard failure, per the screenshot below:

![shard_failure](https://user-images.githubusercontent.com/4459398/72189130-4e122600-33b9-11ea-9c44-f801f3ef0542.png)

Adding a timeout per the following code fixed it:

```
cy.get('[data-test-subj="modal-inspect-close"]').click({ timeout: DEFAULT_TIMEOUT });
```